tests/unit/client/data-access/UlsLanguageRepository.spec.ts

```diff
@@ -1,26 +1,36 @@
 import UlsLanguageTranslationRepository from '@/client/data-access/UlsLanguageTranslationRepository';
+import { StringTMap } from '@/datamodel/LanguageTranslations';
+import MwWindow from '@/client/MwWindow';
 
-function newUlsLanguageRepository( getLanguageNameByCode: ( langCode: string ) => string, langChain: string[] ) {
-	return new UlsLanguageTranslationRepository( getLanguageNameByCode, langChain );
+class MockWbWikibaseContentLanguages {
```

Comment:
This in `MwWindow.ts` would give us the type support for which we went with TS.

Reply:
I agree on this totally, but ts make nearly impossible to access the constructor if we define it as a type at wb. The solutions for that are very hacky and it's just a way to work around ts and not work on the 'real' problem.